modules/libgit2/1.8.1.bcr.1/overlay/src/libgit2/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+
+cc_library(
+    name = "libgit2_hdrs",
+    hdrs = glob(["**/*.h"]),
+    strip_include_prefix = "/src/libgit2",
+    deps = [
+        "//deps/xdiff:xdiff_hdrs",
+        "//include:git2",
+        "//src/util:util_hdrs",
+    ],
+)
+
+cc_library(
+    name = "git2",
+    srcs = glob([
+        "**/*.c",
+        "**/*.h",
+    ]),
+    implementation_deps = [
+        ":libgit2_hdrs",
+        "//deps/llhttp",
+        "//deps/xdiff",
+        "//src/util",
+    ],
+    visibility = ["//:__subpackages__"],
+    deps = ["//include:git2"],
+)
+
+alias(
```

Comment:
Minor nitpick: this target isn't publicly visible, so its name doesn't need to be preserved via an `alias()`.

Instead, the top-level `//:libgit2` alias can just point directly to `//src/libgit2:git2`.

Reply:
Updated!